src/components/SEO.jsx

```diff
@@ -6,6 +6,22 @@
 // The SEO component will add stuff like title, description, image, lang... in the head of the HTML page
 
 import React from 'react';
+import { Script } from 'gatsby';
+
+function getAdThriveScriptUrl(w) {
```

Comment:
Gatsby needs a script URL here, so I pulled this code from the provided Thrive embed script.